pages/index.vue

```diff
@@ -29,17 +31,21 @@ export default Vue.extend({
   },
   head() {
     return {
-      title: "ReplicaZone - Snipcart Demo",
+      title: this.$i18n.t('full_title').toString()
     };
   },
-  async asyncData({ route }) {
+  async asyncData() {
     return {
-      products: Products,
+      products: {}
     };
   },
+
+  mounted() {
+    this.$data.products = this.$i18n.locale == 'fr' ? ProductsFr : Products
```

Comment:
You can use the `onLanguageSwitched` callback to change products dynamically everytime the language is changed https://i18n.nuxtjs.org/callbacks#onlanguageswitched